src/GraphQL/Builders/FieldBuilder.cs

```diff
@@ -58,7 +58,6 @@ protected FieldBuilder(FieldType fieldType)
             {
                 Name = name,
                 ResolvedType = type,
-                Arguments = new QueryArguments(),
```

Comment:
@Shane32 What do you think about thee 2 lines left?

Reply:
No comment.  Seems fine 🤷‍♂️

Reply:
#3275